### PR TITLE
Cleanup code, SEO, navbar improvement. Getting ready for more features.

### PR DESCRIPTION
**Pull request by ChristopherTrimboli**
- Refactor and Cleanup CSS. Use `styles.css` for app specific CSS. In future I will create a `common.css` for common Exokit frontend components. There was lots of unused CSS in this app.

- Comment JS.

- Click away popout-navbar when user clicks outside popout.

- Navbar does not move page content, it overlaps it.

- Exokit logo grows when hovering over it. UX improvement to symbolize you can click it. (did not use library to do grow, just 2 lines of CSS.

- Cleanup/delete dead HTML in app.

- Meta tags for SEO.

- Viewport tag to support mobile users.

- `<body>` is allowed to overflow now. With more than 2 apps this app would not scroll down.

### CODE REVIEW

**Comment by avaer** [comment]
I would hold off on the Meta tags for now.

File: index.html [hunk 1]
```diff
@@ -1,577 +1,194 @@
...
+      document.getElementById('nav-popout').innerHTML = navLinks.map((link, index) => {
+        return `
+          <div>
+            <a href=${link.url} class=${link.active ? "activeNav" : ""}>${link.title}</a>
+          </div>
+        `
+      }).join(`<br/>`)
+
+      // Close nav-popout if user clicks away
+      document.addEventListener("click", (e) => {
```
Comment (avaer) [inline index.html]: This could probably be done with a `blur` event and making the popout focusable (`tabindex=-1`).

Comment (avaer) [reply index.html]: Reason being there are other reasons to close a popup, such as the user pressing tab, or something else popping up in front.

Comment (ChristopherTrimboli) [reply index.html]: 👍

**Review by avaer — APPROVED** [review]

**Comment by avaer** [comment]
👏

**Pull request merged**